### PR TITLE
Update libraries and build config

### PR DESCRIPTION
**Pull request by fitzsia2**
* Target the latest Android version
* Update Android build tools
* Update Facebook SDK
* Update Google Play services
* Update Gradle

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.5.10'
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:4.1.3'
```
Comment (davidbilik) [inline build.gradle:10]: 7.0.2

File: build.gradle [hunk 2]
```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = '1.5.10'
```
Comment (davidbilik) [inline build.gradle]: 1.5.30

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Wed Jan 08 14:15:45 CET 2020
+#Sun Sep 12 16:42:40 CEST 2021
 distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (davidbilik) [inline gradle/wrapper/gradle-wrapper.properties:3]: 7.1.1

File: build.gradle (hunk 1 above)
Comment (fitzsia2) [reply build.gradle:10]: I'm getting an error that SO says is related this line
```
// gradle gradle/bintray/install.gradle
to apply plugin: 'com.github.dcendents.android-maven'
```

How about I update this after migrating to Maven?

File: build.gradle (hunk 2 above)
Comment (fitzsia2) [reply build.gradle]: I amended this change

File: gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (fitzsia2) [reply gradle/wrapper/gradle-wrapper.properties:3]: Like I said above, how about I update this after migrating to Maven?

File: build.gradle (hunk 1 above)
Comment (davidbilik) [reply build.gradle:10]: Why not. So you first do the PR and then rebase this or you want me to merge this and you'll update this in the next PR?

Comment (fitzsia2) [reply build.gradle:10]: Let's merge this one and I'll bump these two versions in the migration PR ✅

**Pull request merged**